Volt/Sandbox/src/Sandbox/Modals/ProjectUpgradeModal.cpp

```diff
@@ -125,7 +918,7 @@ std::pair<std::filesystem::path, Volt::AssetHandle> ProjectUpgradeModal::Deseria
 		streamReader.EnterScope("Metadata");
 
 		resultAssetFilePath = streamReader.ReadKey("filePath", std::string(""));
-		resultAssetHandle = streamReader.ReadKey("assetHandle", 0u);
+		resultAssetHandle = streamReader.ReadKey("assetHandle", uint64_t(0u));
```

Comment:
This could've just be '0ull' instead, but it's fine either way 👍

Reply:
👍